src/core/search/autocompleteresponsetransformer.js

```diff
@@ -6,6 +6,14 @@ import AutoCompleteData, { AutoCompleteResult } from '../models/autocompletedata
  * data structure that our component library and core storage understand.
  */
 export default class AutoCompleteResponseTransformer {
+
+  /**
+   * Converts a universal or vertical autocomplete response from the
+   * core library into an object that the SDK understands.
+   *
+   * @param {import('@yext/answers-core').AutoCompleteResponse} response
+   *  the response passed from the core library
+   */
```

Comment:
Here and below add lets add a `@returns` to indicate the type of what's returned.